CHANGELOG.md

```diff
@@ -0,0 +1,68 @@
+# Change Log
+
+## 5.0.0 26-October-2017
+
+### Breaking Changes
+
+**Deprecations:**  The following colors were moved to Deprecated.
+Use the new name, or import the Deprecated colors module.
+The hex code underneath should not change.
+
+| Nri.Colors.Deprecated function | Equivalent Nri.Colors function    |
+|--------------------------------|-----------------------------------|
+| black                          | gray80                            |
+| blueDark                       | navy                              |
+| blueDark5                      | frost                             |
+| blueDark70                     | navy                              |
+| blueLighter                    | **frost** (see color changes)     |
+| coral                          | red                               |
+| coralLighter                   | redLight                          |
+| coralLightest                  | redLight                          |
+| gray                           | gray75                            |
+| grayDark                       | gray45                            |
+| grayDarker                     | gray45                            |
+| grayLighter                    | gray92                            |
+| grayLightest                   | gray96                            |
+| greenLighter                   | greenLight                        |
+| highlightSolidBlue             | cyan                              |
+| highlightSolidMagenta          | magenta                           |
+| highlightSolidYellow           | yellow                            |
+| linkBlue                       | azure                             |
+| linkBlueDark                   | **azureDark** (see color changes) |
+| linkBlueLight                  | blueLighter                       |
```

Comment:
should this be glacier or frost instead of blueLighter?

Reply:
Totally! Great catch! Thanks!